frontend/public/style/_common.scss

```diff
@@ -488,6 +488,7 @@ $co-external-link-padding-right: 15px;
   [class*='col-'],
   .row {
     vertical-align: middle;
+    position: relative;
```

Comment:
I believe only `.row` needs this addition as [`[class*='col-']` is inheriting it from Bootstrap](https://github.com/twbs/bootstrap/blob/v3.4.1/less/tables.less#L23).

Reply:
thanks, fixed.